### PR TITLE
Add Generic Forwarding documentation

### PR DESCRIPTION
**Pull request by Blinkuu**
**What this PR does**:

This PR adds documentation for the Generic Forwarding feature introduced in https://github.com/grafana/tempo/pull/1775.

**Checklist**
- [x] Documentation added

### CODE REVIEW

File: docs/tempo/website/operations/generic_forwarding.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: Generic forwarding
+weight: 13
+---
+
+# Generic forwarding
+
+Generic forwarding is a feature that allows asynchronous replication of ingested traces. The distributor will write received spans to both the ingester and defined endpoints if enabled. This feature works in a "best-effort" manner, meaning that no retries happen if an error occurs during replication. 
+
+Warning: Generic forwarding does not work retroactively. The distributor will replicate only freshly ingested spans once this feature is enabled.
```
Comment (knylander-grafana) [inline docs/tempo/website/operations/generic_forwarding.md:10]:
```suggestion
>**Warning:** Generic forwarding does not work retroactively. Once enabled, the distributor only replicates freshly ingested spans once this feature.
```

File: docs/tempo/website/operations/generic_forwarding.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+title: Generic forwarding
+weight: 13
+---
+
+# Generic forwarding
+
+Generic forwarding is a feature that allows asynchronous replication of ingested traces. The distributor will write received spans to both the ingester and defined endpoints if enabled. This feature works in a "best-effort" manner, meaning that no retries happen if an error occurs during replication.
```
Comment (knylander-grafana) [inline docs/tempo/website/operations/generic_forwarding.md:8]: Good explanation of the forwarder. I've made a suggestion to tighten the content a little. 

The second sentence says "The distributor"... What does this refer to? We haven't used this term before in this page. 
```suggestion
Generic forwarding allows asynchronous replication of ingested traces. The distributor writes received spans to both the ingester and defined endpoints, if enabled. This feature works in a "best-effort" manner, meaning that no retries happen if an error occurs during replication. 
```

File: docs/tempo/website/operations/generic_forwarding.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+# Generic forwarding
+
+Generic forwarding is a feature that allows asynchronous replication of ingested traces. The distributor will write received spans to both the ingester and defined endpoints if enabled. This feature works in a "best-effort" manner, meaning that no retries happen if an error occurs during replication. 
+
+Warning: Generic forwarding does not work retroactively. The distributor will replicate only freshly ingested spans once this feature is enabled.
+
+## Configuration
+
+Enabling generic forwarding requires the configuration of two sections. First, define a list of forwarders in the `distributor` section. Each forwarder must specify a unique `name`, supported `backend`, and backend-specific configuration. Second, reference these forwarders in the `overrides` section. This allows for fine-grained control over forwarding and makes it possible to enable this feature globally or on a per-tenant basis.
```
Comment (knylander-grafana) [inline docs/tempo/website/operations/generic_forwarding.md:14]: Should this paragraph really be a numbered procedure?

File: docs/tempo/website/operations/generic_forwarding.md (hunk 2 above)
Comment (Blinkuu) [reply docs/tempo/website/operations/generic_forwarding.md:8]: The forwarder is a new subcomponent of the distributor. It wasn't there before.

File: docs/tempo/website/operations/generic_forwarding.md (hunk 3 above)
Comment (Blinkuu) [reply docs/tempo/website/operations/generic_forwarding.md:14]: Hmm, you can't have (2) if you didn't set up (1) in the first place.

File: docs/tempo/website/operations/generic_forwarding.md (hunk 1 above)
Comment (Blinkuu) [reply docs/tempo/website/operations/generic_forwarding.md]: Done

File: docs/tempo/website/operations/generic_forwarding.md (hunk 2 above)
Comment (Blinkuu) [reply docs/tempo/website/operations/generic_forwarding.md]: Applied your suggestion.

**Comment by knylander-grafana** [comment]
@Blinkuu  I wasn't sure if I was supposed to merge or if you would like to, so I haven't merged this PR. Thank you for writing the doc for this!

**Comment by Blinkuu** [comment]
@knylander-grafana I'm not authorized to merge this PR. Perhaps only code owners can do so? Please merge it if you can.

**Pull request merged**